typedargs/annotate.py

```diff
@@ -86,7 +82,7 @@ def get_help(func):
         return help_text
 
     help_text += "\nArguments:\n"
-    for key, info in iteritems(func.metadata.annotated_params):
+    for key, info in iter(func.metadata.annotated_params.items()):
```

Comment:
I think `iter` here is unnecessary since `items()` returns an iterable already.

I didn't realize we used `iteritems` and `viewitems` separately but I never made a distinction between the two and we use them interchangeably.

Reply:
Both `viewitems()` and `iteritems()` map to `.items()` on python 3:
https://python-future.org/_modules/future/utils.html

Reply:
ah. I was using some random PEP doc and just dropped in the equivalents. https://legacy.python.org/dev/peps/pep-0469/ 

Good to know i can just drop the iter part.